Userland/Libraries/LibGfx/ImageFormats/ISOBMFF/JPEG2000Boxes.h

```diff
@@ -99,7 +99,7 @@ struct JPEG2000ColorSpecificationBox final : public Box {
         ICC_Restricted = 2,
```

Comment:
In the commit message: `s/traling/trailing/`

Reply:
😅